app/actions/index.js

```diff
@@ -125,7 +168,7 @@ export const clearRemoteSurveys = () => (dispatch, getState) =>
     type: types.CLEAR_REMOTE_SURVEYS
   });
 
-export const fetchRemoteSurvey = (id, url) => (dispatch, getState) => {
+export const fetchRemoteSurvey = (id, url, target) => (dispatch, getState) => {
```

Comment:
the data sync is triggered when the user taps the survey name after the list of surveys has been fetched. is that not right?

the `syncSurveyData` method could be called in a different place, rather than as part of `fetchRemoteSurvey`.

`onPress` in RemoteSurveyList.js could trigger both `syncSurveyData` and `fetchRemoteSurvey`.

or another approach i'm overlooking?